mllib/src/main/scala/org/apache/spark/ml/feature/HashingTF.scala

```diff
@@ -31,6 +31,12 @@ import org.apache.spark.sql.types.{ArrayType, StructType}
 /**
  * :: Experimental ::
  * Maps a sequence of terms to their term frequencies using the hashing trick.
+ * Current we support two hash algorithms: "murmur3"(default) and "native".
+ * "murmur3" calculates a hash code value for the term object using
+ * Austin Appleby's MurmurHash 3 algorithm (MurmurHash3_x86_32);
+ * "native" calculates the hash code value using the native Scala implementation.
+ * In Spark 1.6 and earlier, "native" is the default hash algorithm;
+ * after Spark 2.0, we use "murmur3" as the default one.
```

Comment:
@MLnick Thanks for your comments. It has been recorded at [ML/MLlib behavior changes in 2.0](https://issues.apache.org/jira/browse/SPARK-13448) for documentation.